src/ngraph/op/fused/group_conv.cpp

```diff
@@ -130,39 +168,32 @@ shared_ptr<Node> op::GroupConvolution::copy_with_new_args(const NodeVector& new_
 NodeVector op::GroupConvolution::decompose_op() const
 {
     auto data = input_value(0);
+    auto data_shape = get_input_shape(0);
```

Comment:
In general, for now, we are not worrying about making `decompose_op` work with dynamic shapes, but be aware that in the future this will be a problem.

Reply:
I removed this variable because it was an unused leftover from draft implementation.
Thanks for mentioning it.